docs/modules/deployment/pages/backup/debian-ubuntu/backup-debian-ubuntu.adoc

```diff
@@ -0,0 +1,34 @@
+[[backup-debian-ubuntu]]
+.Stop OpenNMS {page-component-title} service
+[source, console]
+----
+sudo systemctl stop opennms
+----
+
+.Create backup folder
+[source, console]
+----
+sudo mkdir /backup
```

Comment:
What do you think about suggesting `/tmp/backup` as a target instead?

Reply:
`/tmp` is always dangerous since a reboot deletes all data there. But we could change this note https://github.com/OpenNMS/opennms/pull/4484/files#diff-4c17aea04c253168f7264ce259f6c1a69636752d837b49d6c881bfe9247f30c5R7 and warn about `/tmp` usage. Wdyt?

Reply:
Fixed with https://github.com/OpenNMS/opennms/pull/4484/commits/a19e20a27831c146c9b7355bb8f9c801fc2914e3